src/ExpenseTrackerGrupo2/Persistence/Database/DbInitializer.cs

```diff
@@ -0,0 +1,6 @@
+namespace ExpenseTrackerGrupo2.Persistence.Database;
+
+public class DbInitializer
+{
+    // TODO: Complete according the sql scripts
```

Comment:
We set the database through docker-compose. I dont know if is still needed. What do you think?